app/controllers/books_controller.rb

```diff
@@ -10,6 +11,7 @@ def show
 
   def new
     @book = Book.new
+    authorize @book
```

Comment:
why to authorize here when you have `before_action` method which is called before each action?